app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="sign-container">
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+  <%= f.label :name %><br />
+  <%= f.text_field :name, autofocus: true, autocomplete: "text" %>
+</div>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class:"btn-sign"  %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] Please make sure the sign-up page's design adheres to the wireframe. You can completely customize your visual design based on the requirements, but you should maintain the wireframe. 👍️

### wireframe :point_down: 

![Screenshot from 2023-09-07 15-07-12](https://github.com/binodbhusal/BlogApp/assets/89228743/d83d8b50-4f19-4acb-a7e9-ffd3913c1862)

### Note: :point_down: 
This wireframe, as you can see, is for the sign-in page, but please make sure to adapt the same design for the sign-up page. The sign-up form contains more inputs than the others, but other than that, they should have a similar layout. :ok_hand: 😊